[bitnami/minio] .Values.networkPolicy.extraFromClauses wrong default type

### Name and Version

bitnami/minio 12.1.10

### What architecture are you using?

None

### What steps will reproduce the bug?

1. Create sample Helm Chart with `helm create sample`
2. Include Bitnami/Minio in dependencies of `sample`
    ```yaml
    dependencies:
      - name: minio
        repository: https://charts.bitnami.com/bitnami
        version: "12.x.x"
    ```
4. Put in values.yaml of `sample` chart this:
    ```yaml
    minio:
      networkPolicy:
        enabled: true
        extraFromClauses:
          - podSelector: {}
    ```
5.  Execute `helm lint sample/`. Any of `helm template` or `helm install` also produce the same bug.

### Are you using any custom parameters or values?

_No response_

### What is the expected behavior?

I expect to see successful lint without warnings:
```console
==> Linting .
[INFO] Chart.yaml: icon is recommended

1 chart(s) linted, 0 chart(s) failed
```

 The fix is to set in default values.yaml of Bitnami/Minio `list` instead of `map` in `networkPolicy.extraFromClauses` as below:
```yaml
networkPolicy:
  extraFromClauses: []
```
It must be a `list`, even the example shows it at line 869:
```yaml
  ## Example
  ## extraFromClauses:
  ## - podSelector:
  ##     matchLabels:
  ##       a: b
```

### What do you see instead?

```console
coalesce.go:220: warning: cannot overwrite table with non table for sample.minio.networkPolicy.extraFromClauses (map[])
coalesce.go:220: warning: cannot overwrite table with non table for sample.minio.networkPolicy.extraFromClauses (map[])
coalesce.go:220: warning: cannot overwrite table with non table for sample.minio.networkPolicy.extraFromClauses (map[])
coalesce.go:220: warning: cannot overwrite table with non table for sample.minio.networkPolicy.extraFromClauses (map[])
```

### Additional information

_No response_

--- a/bitnami/minio/Chart.yaml
+++ b/bitnami/minio/Chart.yaml
@@ -25,4 +25,4 @@ name: minio
 sources:
   - https://github.com/bitnami/containers/tree/main/bitnami/minio
   - https://min.io
-version: 12.1.10
+version: 12.1.11

--- a/bitnami/minio/README.md
+++ b/bitnami/minio/README.md
@@ -235,7 +235,7 @@ The command removes all the Kubernetes components associated with the chart and 
 | `apiIngress.extraRules`            | Additional rules to be covered with this ingress record                                                                          | `[]`                     |
 | `networkPolicy.enabled`            | Enable the default NetworkPolicy policy                                                                                          | `false`                  |
 | `networkPolicy.allowExternal`      | Don't require client label for connections                                                                                       | `true`                   |
-| `networkPolicy.extraFromClauses`   | Allows to add extra 'from' clauses to the NetworkPolicy                                                                          | `{}`                     |
+| `networkPolicy.extraFromClauses`   | Allows to add extra 'from' clauses to the NetworkPolicy                                                                          | `[]`                     |
 
 ### Persistence parameters
 

--- a/bitnami/minio/values.yaml
+++ b/bitnami/minio/values.yaml
@@ -865,7 +865,7 @@ networkPolicy:
   ##
   allowExternal: true
   ## @param networkPolicy.extraFromClauses Allows to add extra 'from' clauses to the NetworkPolicy
-  extraFromClauses: {}
+  extraFromClauses: []
   ## Example
   ## extraFromClauses:
   ## - podSelector:

